packages/create-svelte/shared/+vitest-typescript/vitest.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite';
+import viteConfig from './vite.config';
+
+export default defineConfig({
```

Comment:
why do we need a separate `vitest.config.js` vs just using the existing `vite.config.js`?